src/state.rs

```diff
@@ -295,6 +299,26 @@ impl<'a, S: ReadonlyStorage> ReadonlyBalancesImpl<'a, S> {
     }
 }
 
+pub struct RevokedPemits;
+
+impl RevokedPemits {
+    pub fn is_permit_revoked(
+        storgae: &dyn Storage,
+        account: &HumanAddr,
+        permit_name: &str,
+    ) -> bool {
+        let storage_key = account.to_string() + permit_name;
```

Comment:
While showing Sienna this code segment, i noticed that it should also add a constant prefix to this, just like `PREFIX_VIEW_KEY` is used, for example, but here we can go simpler by just writing

```rust
let storage_key = REVOKED_PERMIT_KEY.to_string() + &account.to_string() + permit_name;
```

Reply:
Yup, thanks.